scripts/dev_tools/sync_issues.py

```
@@ -0,0 +1,155 @@
+ISSUES_DIR = Path(__file__).parent.parent.parent / "issues"
+GITHUB_API_BASE = "https://api.github.com"
+
+
+def get_github_token():
+    """Get GitHub token from env var or gh CLI."""
+    token = os.getenv("GITHUB_TOKEN")
+    if token:
+        return token
+    try:
+        result = subprocess.run(
+            ["gh", "auth", "token"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)